src/Core/src/Platform/Android/Resources/values/styles.xml

```diff
@@ -16,6 +16,7 @@
         <item name="checkboxStyle">@style/MauiCheckBox</item>
         <item name="android:textAllCaps">false</item>
         <item name="alertDialogTheme">@style/MauiAlertDialogTheme</item>
+        <item name="materialAlertDialogTheme">@style/MauiAlertDialogTheme</item>
```

Comment:
I see we use the same material dilog style for both material and app compat, but still might be worth not changing too much in net9.

Maybe for net10 we switch all the pickers to material? But that is a different issue.